### PR TITLE
Added Dockerfile for E-Commerce Worker service

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-5862

### CODE REVIEW

File: docker/plays/ecomworker.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- name: Deploy edX Ecommerce Worker
+  hosts: all
+  become: True
+  gather_facts: True
+  vars:
+    ENABLE_DATADOG: False
+    ENABLE_SPLUNKFORWARDER: False
+    ENABLE_NEWRELIC: False
+  roles:
+    - aws
```
Comment (e0d) [inline docker/plays/ecomworker.yml]: Should be:

```
roles:
  - common_vars
  - docker
  - ecomworker
  ...
```

Comment (clintonb) [reply docker/plays/ecomworker.yml]: Updated.

**Comment by e0d** [comment]
:+1:

**Review by e0d — APPROVED** [review]
Looks good.

**Comment by clintonb** [comment]
We will revisit this in the future if/when we move this into the Docker-based devstack.

**Pull request closed without merging**